Core/H5PSymfony.php

```diff
@@ -214,11 +227,10 @@ public function setInfoMessage($message)
      */
     public function getMessages($type)
     {
-        if (!$this->session->getFlashBag()->has($type)) {
+        if ($this->session === null || !$this->session->getFlashBag()->has($type)) {
```

Comment:
replace this row is better 😄 

```suggestion
        if (!$this->session || !$this->session->getFlashBag()->has($type)) {
```